settings.gradle

```diff
@@ -6,16 +6,31 @@ include 'kieker-monitoring'
 include 'kieker-analysis'
 include 'kieker-model'
 include 'kieker-tools'
+// tools
+include 'kieker-tools:restructuring'
 include 'kieker-tools:behavior-analysis'
+include 'kieker-tools:cmi'
 include 'kieker-tools:collector'
 include 'kieker-tools:convert-logging-timestamp'
+include 'kieker-tools:dar'
+include 'kieker-tools:delta'
+include 'kieker-tools:fxca'
 include 'kieker-tools:log-replayer'
+include 'kieker-tools:maa'
+include 'kieker-tools:mktable'
+include 'kieker-tools:mop'
+include 'kieker-tools:mt'
+include 'kieker-tools:mvis'
 include 'kieker-tools:opad'
+include 'kieker-tools:pp-static-log'
+include 'kieker-tools:relabel'
 include 'kieker-tools:resource-monitor'
-//include 'kieker-tools:runtime-analysis'
+include 'kieker-tools:rewrite-log-entries'
+include 'kieker-tools:runtime-analysis'
+include 'kieker-tools:sar'
 include 'kieker-tools:trace-analysis'
-include 'kieker-tools:trace-analysis-new'
```

Comment:
Why was the order changed here?

Reply:
I don't know. Could have been an automatic ordering.